extensions/standard-processors/processors/PutTCP.cpp

```diff
@@ -114,6 +114,21 @@ void PutTCP::initialize() {
 
 void PutTCP::notifyStop() {}
 
+namespace {
+asio::ssl::context getSslContext(const std::shared_ptr<controllers::SSLContextService>& ssl_context_service) {
+  gsl_Expects(ssl_context_service);
+  asio::ssl::context ssl_context(asio::ssl::context::sslv23);
```

Comment:
Sure, I've restricted to only allow TLS 1.2 or newer versions in https://github.com/apache/nifi-minifi-cpp/pull/1457/commits/653894f83c9464edda7d5238408937074f51c34c#diff-a62ddb0368e1813dff2a90fe7433e148bfc7b21b7d7ad9c8a066fd040d209ac9R120

Reply:
It looks like TLS 1.2 only. TLS 1.3 should be preferred.

Reply:
https://stackoverflow.com/a/47097088

Reply:
You are right, and the test didn't catch it because our libressl doesnt have TLSv1.3 yet.
I updated it so when we update our SSL version it should work.
https://github.com/apache/nifi-minifi-cpp/pull/1457/commits/1a62da1c25a15ced0c664fe1128cdb48792ba331